Doc/library/webbrowser.rst

```diff
@@ -171,9 +171,6 @@ Notes:
    Removed browsers include Grail, Mosaic, Netscape, Galeon,
```

Comment:
Can't reach it, but as previously discussed I would suggest adding the missing `.. versionchanged:: 3.2` above the 3.3 that `MacOSXOSAScript` was added and used by default instead of `MacOSX` for Mac. This retains the remaining relevant bit from the `deprecated-removed` message, in a more useful fashion for readers.

E.g. something like

```rst
.. versionadded:: 3.2
   A new :class:`!MacOSXOSAScript` class has been added
   and is used on Mac instead of the previous :class:`!MacOSX` class.
   This adds support for opening browsers not currently set as the OS default.
```

Reply:
Thanks, updated!